tests/testthat/comparison_data/energy/L117.RsrcCurves_EJ_R_tradbio.csv

```diff
@@ -0,0 +1,60 @@
+# ../energy-data/level1//L117.RsrcCurves_EJ_R_tradbio.csv
+# Written by LA117.tradbio.R
+# Fri Mar 31 15:50:41 2017
+# " Traditional biomass resources by GCAM region "
+# " Unit = EJ "
```

Comment:
Ben, @bpbond , technically there should be units of 1975$/GJ on the extraction cost column as well as EJ on the available column.  But that would mean this file would be different.  I'm sitting with Russell and we were wondering if he should just add the unit, or we should add it as an issue, or even add it to code for if_new_behavior or whatever you called it.  Otherwise this looks good